src/material/core/common-behaviors/common-module.ts

```diff
@@ -6,9 +6,16 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {NgModule, InjectionToken, Optional, Inject, isDevMode} from '@angular/core';
+import {NgModule, InjectionToken, Optional, Inject, isDevMode, Version} from '@angular/core';
```

Comment:
This should be the `VERSION` from `@angular/material/core' correct?  Currently, the check attempts to ensure that the full version of CDK and Angular core are matching.

Reply:
Nevermind, I misread/was mistaken.